doc/stdenv/meta.chapter.md

```diff
@@ -86,6 +86,17 @@ meta.platforms = lib.platforms.linux;
 
 Attribute Set `lib.platforms` defines [various common lists](https://github.com/NixOS/nixpkgs/blob/master/lib/systems/doubles.nix) of platforms types.
 
+### `badPlatforms` {#var-meta-badPlatforms}
+
+The list of Nix platform types on which the package is known not to be buildable.  Hydra will never create prebuilt binaries for these platform types, even if they are in `meta.platforms`.  In general it is preferable to set `meta.platforms = lib.platforms.all` and then exclude any platforms on which the package is known not to build.  For example, a package which requires dynamic linking and cannot be linked statically could use this:
+
+```nix
+meta.platforms = lib.platforms.all;
+meta.badPlatforms = [ lib.systems.inspect.patterns.isStatic ];
+```
```

Comment:
Ideally we'd have a complete example that we could refer to. We have to expect readers to drop into any piece of documentation with barely any contextual knowledge of Nix or Nixpkgs idioms or inner workings. While this snippet is enough if you know that it would apply in a `mkDerivation` call, it gets really confusing if you don't. Maybe in the example introduction we can refer and link to a full example, outlining how this snippet would fit in and which effect it would have.